.packit.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# docs: https://packit.dev/docs/configuration/
+upstream_package_name: rpmlint
+specfile_path: .packit/rpmlint.spec
+
+create_archive: ["./packit/generate-tarball.sh"]
```

Comment:
```suggestion
create_tarball_command: ["./packit/generate-tarball.sh"]
```

Reply:
I'd advocate for the `create-archive` action: https://packit.dev/docs/actions/
```
actions:
  create-archive: "./packit/generate-tarball.sh"
```